src/transformers/models/owlvit/feature_extraction_owlvit.py

```diff
@@ -32,14 +32,58 @@
 logger = logging.get_logger(__name__)
 
 
+# Copied from transformers.models.detr.feature_extraction_detr.center_to_corners_format
```

Comment:
This is making the method less readable.

Reply:
@sgugger do you mean we shouldn't use copied from statements?

Reply:
Or better, fix the function in DETR so it uses the same variable names? Can be done in a followup PR.